cmd/context/show.go

```diff
@@ -62,8 +68,7 @@ func Show() *cobra.Command {
 		},
 	}
 	cmd.Flags().StringVarP(&output, "output", "o", "json", "output format. One of: ['json', 'yaml']")
-
-	return cmd
+    cmd.Flags().BoolVar(&includeToken, "include-token", false, "include the token in the output")	return cmd
```

Comment:
a bad format here. does not compile
```suggestion
    cmd.Flags().BoolVar(&includeToken, "include-token", false, "include the token in the output")
    return cmd
```

Reply:
Sorry! Resolved now